protocol/contracts/libraries/LibDibbler.sol

```diff
@@ -43,55 +43,44 @@ library LibDibbler {
 
     /**
      * @param beans The number of Beans to Sow
-     * @param morningYield FIXME
+     * @param _morningTemperature FIXME
      * @param account The account sowing Beans
      * @dev 
      * 
      * ## Above Peg 
      * 
-     * | t   | pods  | soil                                   | yield                          | maxYield     |
+     * | t   | pods  | soil                                   | temperature                          | maxTemperature     |
      * |-----|-------|----------------------------------------|--------------------------------|--------------|
      * | 0   | 500e6 | ~6683e6 (500e6 * (1 + 1250%)/(1+1%))   | 1e6 (1%)                       | 1250 (1250%) |
      * | 12  | 500e6 | ~1507e6 (500e6 * (1 + 1250%)/(1+348%)) | 348.75e6 (27.9% * 1250 * 1e6)  | 1250         |
      * | 300 | 500e6 |  500e6 (500e6 * (1 + 1250%)/(1+1250%)) | 1250e6                         | 1250         |
      * 
      * ## Below Peg
      * 
-     * | t   | pods                            | soil  | yield                         | maxYield     |
+     * | t   | pods                            | soil  | temperature                         | maxTemperature     |
      * |-----|---------------------------------|-------|-------------------------------|--------------|
      * | 0   | 505e6 (500e6 * (1+1%))          | 500e6 | 1e6 (1%)                      | 1250 (1250%) |
      * | 12  | 2243.75e6 (500e6 * (1+348.75%)) | 500e6 | 348.75e6 (27.9% * 1250 * 1e6) | 1250         |
      * | 300 | 6750e6 (500e6 * (1+1250%))      | 500e6 | 1250e6                        | 1250         |
-     * 
-     * Yield is floored at 1%.
-     * the amount of soil changes as a function of the morning auction;
-     * soil consumed increases as dutch auction passes
-     * t = 0   -> tons of soil
-     * t = 300 -> however much soil to get fixed number of pods at current temperature
-     * soil subtracted is thus scaled down:
-     * soilSubtracted = s.f.soil * SoilSowed/totalSoilAbovePeg
-     * soilSubtracted = s.f.soil * SoilSowed/(s.f.soil * ((1 + s.w.yield) /(1 + yield())))
-     * soilSubtracted = Amt * (1 + yield())/(1+ s.w.yield) 
-     * soilSubtracted = pods/(1+ s.w.yield) 
      */
-    function sow(uint256 beans, uint256 morningYield, address account) internal returns (uint256) {
+    function sow(uint256 beans, uint256 _morningTemperature, address account) internal returns (uint256) {
         AppStorage storage s = LibAppStorage.diamondStorage();
         
         uint256 pods;
-        uint256 maxYield = uint256(s.w.yield).mul(YIELD_PRECISION);
+        uint256 maxTemperature = uint256(s.w.t).mul(TEMPERATURE_PRECISION);
 
         // Above peg: FIXME
         if (s.season.abovePeg) {
             // amount sown is rounded up, because 
-            // 1: yield is rounded down.
+            // 1: temperature is rounded down.
```

Comment:
should put this note into scaleSoilDown? since its not obvious what is happening here